lte/gateway/c/session_manager/SetMessageManagerHandler.cpp

```diff
@@ -190,10 +242,116 @@ void SetMessageManagerHandler::initiate_release_session(
    * print the fatal error and move on
    */
   if (update_success) {
-    MLOG(MINFO)
+    MLOG(MDEBUG)
         << "Successfully released and updated SessionStore of subscriber"
         << imsi;
   }
 }
 
+/* This function called when any specific  session moved to
+ * idle state
+ */
+void SetMessageManagerHandler::pdu_session_inactive(
+    const SetSmNotificationContext& notif,
+    std::function<void(Status, SmContextVoid)> response_callback) {
+  // extract values from proto
+  uint32_t pdu_id  = notif.rat_specific_notification().pdu_session_id();
+  std::string imsi = notif.common_context().sid().id();

```

Comment:
```
[cpplint] reported by reviewdog 🐶
 Add #include <string> for string  [build/include_what_you_use] [4]
```

Reply:
Included the header file <string> as suggeseted.